main.go

```diff
@@ -231,9 +250,13 @@ func serve(logger log.Logger, addr string, writers []writer, readers []reader) e
 			}(w)
 		}
 		wg.Wait()
+		duration := time.Since(begin).Seconds()
```

Comment:
What's the resolution of this ? Does it go down to milliseconds (floating point number)? We'd need that if it doesn't do it.

Reply:
It is a floating point seconds value. So we are good